modules/video/src/lkpyramid.cpp

```diff
@@ -1487,17 +1487,15 @@ getRTMatrix( const Point2f* a, const Point2f* b,
 
 }
 
-cv::Mat cv::estimateRigidTransform( InputArray src1, InputArray src2, bool fullAffine )
+cv::Mat cv::estimateRigidTransform( InputArray src1, InputArray src2, bool fullAffine, int RANSAC_MAX_ITERS, double RANSAC_GOOD_RATIO,
+                                    int RANSAC_SIZE0)
 {
     CV_INSTRUMENT_REGION()
 
     Mat M(2, 3, CV_64F), A = src1.getMat(), B = src2.getMat();
 
     const int COUNT = 15;
     const int WIDTH = 160, HEIGHT = 120;
-    const int RANSAC_MAX_ITERS = 500;
```

Comment:
Since you now pass parameters instead of using constants, there should be some code added to check the parameters for reasonable value ranges